client/src/utils/timeFormat.js

```diff
@@ -14,6 +24,27 @@ export function formatTime(timeDateValue) {
   });
 }
 
+/**
+ * Create a string representation of date and time using internal standard
+ * @param {Date} dateTime - A date object
+ * @returns {string}
+ */
 export function formatDateTime(timeDateValue) {
   return `${formatDate(timeDateValue)} ${formatTime(timeDateValue)}`;
 }
+
+/**
+ * A standard-format date string or UNIX timestamp
+ * @typedef {string|number} dateTime
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date
+ */
+/**
+ * Create a string representation of date/time using internal standard
+ * @param {dateTime} dateTimeValue - A single value date-time representation
+ * @returns {string}
+ */
+export function formatDateTimeFromValue(dateTimeValue) {
+  const date = new Date(dateTimeValue);
+
+  return formatDateTime(date);
+}
```

Comment:
Using this would reduce the need for creating a `Date` every time before calling `formatDateTime`.

![All The Dates](https://user-images.githubusercontent.com/62723358/113943403-40b83700-97c8-11eb-9da0-9055e7291cf1.png)

Reply:
cool. 👍 . is the plan to change the use from `formatDateTime` to `formatDateTimeFromValue` in this PR? or in the future?

Reply:
Up to y'all. This `quick/` branch has no requirements. I just noticed, while reviewing my work for side effects, it is possible to do so.

Reply:
Sal, you've approved, so I assume this is not something to do now.

Because this is a `quick/` branch that is off the roadmap, I'd like to minimize operational changes.

_I'm trying to support improvements to quality of life, not make broad changes to the codebase._

Reply:
follow on ticket sounds good 👍 

> Because this is a quick/ branch that is off the roadmap, I'd like to minimize operational changes.

totally agree.